app/src/main/java/com/anzelmas/quizzando/QuizQuestionActivity.kt

```diff
@@ -14,6 +15,145 @@ import androidx.databinding.DataBindingUtil
 import androidx.databinding.ViewDataBinding
 import kotlinx.android.synthetic.main.activity_quiz_question.*
 
+class QuizQuestionActivity : AppCompatActivity(), View.OnClickListener {
+
+
+    private var currentPosition: Int = 1
+    private var questionList: ArrayList<Question>? = null
+    private var selectedOption: Int = 0
+    private var correctAnswersNumber: Int = 0
+    private var userName: String? = null
+
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_quiz_question)
+
+        userName = intent.getStringExtra(Constants.USER_NAME)
+
+        questionList = Constants.getQuestions()
+//        Log.i("Question size", "${questionList!!.size}")
+
+        putQuestion()
+
+        optionOne_TextView.setOnClickListener(this)
+        optionTwo_textView.setOnClickListener(this)
+        optionThree_TextView.setOnClickListener(this)
+        optionFour_TextView.setOnClickListener(this)
+        submit_Button.setOnClickListener(this)
+    }
+
+    private fun putQuestion() {
+        val question = questionList!![currentPosition - 1]
```

Comment:
Try avoid using `!!` and find a safe way to call it, in other case if `questionList` will be `null`, your app is going to crash - we don't want that scenario.